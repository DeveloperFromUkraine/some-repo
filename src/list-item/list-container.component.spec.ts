import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListContainerComponent } from "index";
import { ComponentTest } from "../../test/test-bed/component";

describe('ListContainerComponent', () => {
    let fixture: ComponentFixture<ListContainerComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [ListContainerComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListContainerComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});