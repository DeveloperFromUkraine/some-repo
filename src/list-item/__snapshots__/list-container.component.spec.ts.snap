// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Container should match snapshot 1`] = `
<ign-list-container>
  
</ign-list-container>
`;
